specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/examples/AgentPoolsCreate_Update.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "api-version": "2019-02-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "managedClusterName": "clustername1",
+    "agentPoolName":"agentpool1",
+    "parameters": {
+      "properties": {
+        "orchestratorVersion": "",
+        "count": 3,
+        "vmSize": "Standard_DS1_v2",
+        "osType": "Linux"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/managedClusters/clustername1/agentPools/agentpool1",
```

Comment:
Please fix the example, the name segment in id does not match the value in name property. "type" property is missing from the response.